src/popup/pages/Invite.vue

```diff
@@ -91,9 +91,14 @@ export default defineComponent({
 
       try {
         const sdk = await getSdk();
-        await sdk.spend(formModel.value.amount, publicKey, {
-          denomination: AmountFormatter.AE_AMOUNT_FORMATS.AE,
-        });
+        await sdk.spend(
+          formModel.value.amount || 0,
+          publicKey,
+          /* TODO - denomination is rejected by TS types
+           * but by the structure looks like it should be allowed of buildTx
+           */
+          { denomination: AmountFormatter.AE_AMOUNT_FORMATS.AE },
```

Comment:
Could be a bug from SDK! We need to create an issue in SDK repo.

Reply:
@davidyuk maybe you can help with this?

Reply:
I think it is because you have `strictFunctionTypes` enabled.

Reply:
https://github.com/aeternity/superhero-wallet/pull/2140/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0R15